eng/build.sh

```diff
@@ -95,6 +152,11 @@ while [[ $# > 0 ]]; do
   opt="$(echo "${1/#--/-}" | awk '{print tolower($0)}')"
 
   if [[ $firstArgumentChecked -eq 0 && $opt =~ ^[a-zA-Z.+]+$ ]]; then
+    if [ $opt == "help" ]; then
+      showSubsetHelp
+      exit 0
+    fi
+
     arguments="$arguments /p:Subset=$1"
```

Comment:
Now that you're touching this, should `/p:Subset=$1` be `/p:Subset=$opt`? don't know if that'll make a difference, but it is weird to check if $opt matches the regex, if opt is == help and then use $1.

Reply:
I think you're right and using `$opt` would make the code look cleaner. Just before making this change, the arguments for MSBuild are not case-sensitive right? In the sense that `/p:Subset=CLR` would work just fine like `/p:Subset=clr`?

Reply:
IIRC it is case sensitive in Unix, but we lowercase the subsets before processing them:

https://github.com/dotnet/runtime/blob/8f1c72d7c414b88874949c4a0ef9f65c1c7e05d3/eng/Subsets.props#L46